packages/graphback-datasync/src/providers/DatasyncMongoDBDataProvider.ts

```diff
@@ -11,6 +11,15 @@ export class DataSyncMongoDBDataProvider<Type = any, GraphbackContext = any> ext
 
   public constructor(baseType: GraphQLObjectType, client: any) {
     super(baseType, client);
+    applyIndexes([
+      {
+        key: {
+          _deleted: 1
+        }
+      }
+    ],this.db.collection(this.collectionName)).catch((e: any) => {
```

Comment:
Seems like the `applyIndexes` function already catches the error and does not re-thrown: https://github.com/aerogear/graphback/pull/1276/files#diff-0310a0356ee20ce16b7b5d3f6d8d6420R16, maybe this catch  clause here is redudant.